src/components/topics-list.js

```diff
@@ -29,9 +29,9 @@ export class TopicsList {
         name: 'Process',
         topics: [
           {
-            title: 'Recieve food',
+            title: 'Receive food',
             form:
-              'https://api.forms.platforms.qld.gov.au/fesrqwsyzlbtegd/recievefood',
+              'https://api.forms.platforms.qld.gov.au/fesrqwsyzlbtegd/receivefood',
```

Comment:
should this be setting
{this.submissionInfo.projectID}
${this.submissionInfo.formID}